splinepy/microstructure/tiles/double_lattice.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 
 from splinepy.bezier import Bezier
-from splinepy.microstructure.tiles.tilebase import TileBase
+from splinepy.microstructure.tiles.tile_base import TileBase
 
 
-class DoubleLatticeTile(TileBase):
+class DoubleLattice(TileBase):
```

Comment:
Hard to guess dimension of this one. Also, is there a better name? The name [DoubleLattice](https://en.wikipedia.org/wiki/Double_lattice) can be misleading. This is more like `XBox`, but this name may be taken by another group.

Reply:
Plus, please double check after copy&paste. Documentation for `__init__` is same as cross and wrong

Reply:
See previous comment, also updated docs